stub_uploader/build_wheel.py

```diff
@@ -254,6 +272,8 @@ def generate_long_description(
                 stub_distribution=metadata.stub_distribution
             )
         )
+    elif metadata.partial:
```

Comment:
I think it might be a good idea to still have the "partial" comment, even if the package is outdated, if only for consistency reasons. Of course, it's a bit weird to ask the user to contribute to outdated packages, but I don't think it's too big of a deal.

Reply:
```suggestion
    if metadata.partial:
```